dashboards/kubernetes_capacity_planning/KubernetesCapacityPlanning--2022-09-08T07_48_25.json

```diff
@@ -0,0 +1 @@
:"sum:kubernetes.memory.usage{kube_replica_set:*,$kube_namespace,$kube_cluster_name,$env,$scope} by {kube_replica_set}/sum:kubernetes.memory.requests{kube_replica_set:*,$kube_namespace,$kube_cluster_name,$env,$scope} by {kube_replica_set}*100","style":{"palette":"dog_classic"}}]},"layout":{"x":0,"y":15,"width":6,"height":4}},{"id":7775606387220348,"definition":{"title":"Overprovisioned Replicaset","title_size":"16","title_align":"left","type":"scatterplot","requests":{"table":{"formulas":[{"alias":"","dimension":"x","formula":"query2"},{"formula":"query1 / query2 * 100","dimension":"y"}],"response_format":"scalar","queries":[{"query":"sum:kubernetes.memory.requests{kube_replica_set:*,$kube_namespace,$kube_cluster_name,$env,$scope} by {kube_replica_set,kube_cluster_name}","data_source":"metrics","name":"query2","aggregator":"avg"},{"query":"sum:kubernetes.memory.usage{kube_replica_set:*,$kube_namespace,$kube_cluster_name,$env,$scope} by {kube_replica_set,kube_cluster_name}","data_source":"metrics","name":"query1","aggregator":"avg"}]}},"xaxis":{"min":"0"},"yaxis":{"min":"0","max":"200"},"color_by_groups":["kube_cluster_name"]},"layout":{"x":6,"y":15,"width":6,"height":4}},{"id":6338333826598052,"definition":{"title":"Overprovisioned Pods","title_size":"16","title_align":"left","show_legend":false,"type":"heatmap","yaxis":{"min":"0","max":"200"},"requests":[{"q":"sum:kubernetes.memory.usage{$kube_namespace,$kube_cluster_name,$env,$scope} by {pod_name}/sum:kubernetes.memory.requests{$kube_namespace,$kube_cluster_name,$env,$scope} by {pod_name}*100","style":{"palette":"dog_classic"}}]},"layout":{"x":0,"y":19,"width":6,"height":4}},{"id":585478333514986,"definition":{"title":"Overprovisioned Pods","title_size":"16","title_align":"left","type":"scatterplot","requests":{"table":{"formulas":[{"alias":"","dimension":"x","formula":"query2"},{"formula":"query1 / query2 * 100","dimension":"y"}],"response_format":"scalar","queries":[{"query":"sum:kubernetes.memory.requests{$kube_namespace,$kube_cluster_name,$env,$scope} by {pod_name,kube_cluster_name}","data_source":"metrics","name":"query2","aggregator":"avg"},{"query":"sum:kubernetes.memory.usage{$kube_namespace,$kube_cluster_name,$env,$scope} by {pod_name,kube_cluster_name}","data_source":"metrics","name":"query1","aggregator":"avg"}]}},"xaxis":{"include_zero":true,"scale":"linear","min":"auto","max":"auto"},"yaxis":{"min":"0","max":"200"},"color_by_groups":["kube_cluster_name"]},"layout":{"x":6,"y":19,"width":6,"height":4}}]},"layout":{"x":0,"y":37,"width":12,"height":1}}],"template_variables":[{"name":"scope","prefix":"*","available_values":[],"default":"*"},{"name":"env","prefix":"env","available_values":[],"default":"*"},{"name":"kube_cluster_name","prefix":"kube_cluster_name","available_values":[],"default":"*"},{"name":"kube_namespace","prefix":"kube_namespace","available_values":[],"default":"*"}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"c3q-j5n-nw5"}
```

Comment:
Remove datadog_demo_keep from description and add some description to the dashboard with some links to our Docs or public documentation about Kubernetes.

I've done something like this:
`## Kubernetes Capacity Planning
Use this Dashboard to understand the requested CPU/Memory for pods and what is really used

Some resources:
- [Datadog Infra > Kubernetes](https://docs.datadoghq.com/containers/kubernetes/#overview)
- [Kubernetes patterns: capacity planning](https://www.cncf.io/blog/2019/10/08/kubernetes-patterns-capacity-planning/)
- [Kubernetes scaling](https://medium.com/pareture/kubernetes-scaling-capacity-and-resource-planning-in-complex-clusters-97a6105b43a4)
`